### ISSUE TITLE
Publish on Snapcraft.io

### ISSUE THREAD

**Issue by Leniwcowaty** [Mastermindzh/tidal-hifi#493]
Hi there!
Would it be possible to release this on Snapcraft.io? As far as I know, there's a way to automate it with GitHub actions - at least that's what I got from Snapcrafters members on Ubuntu Summit.
I can ask them for any contact information to provide, I will also be attending a workshop led by them on this exact topic.
Cheers man!

**Comment by Mastermindzh** [comment]
I'm certainly willing to try, but I just got this notice served to me:

![image](https://github.com/user-attachments/assets/53a1c7fb-0edf-4d0a-815d-9eed7cffb80e)

Want me to wait until you had the workshop and let you do the snapcraft.yml 😄 ?

**Comment by Leniwcowaty** [comment]
Nah, do your thing, I can wait 30 days 😄 I couldn't attend the workshop sadly

**Comment by Mastermindzh** [comment]
Alright, I'll work on this next time I have some time available :)

**Comment by Mastermindzh** [comment]
Release [6.3.0-Mavy](https://github.com/Mastermindzh/tidal-hifi/releases/tag/6.3.0-Mavy) should address this issue.
Here's the link: https://snapcraft.io/tidal-hifi

Enjoy!


PS: in reality an earlier release already solved this but I forgot to close the issue 😄.
